obspy/core/tests/test_waveform_plugins.py

```diff
@@ -256,13 +269,18 @@ def test_read_thread_safe(self):
         be all the same.
         """
         data = np.arange(0, 500)
-        start = UTCDateTime(2009, 1, 13, 12, 1, 2, 999000)
         formats = _get_default_eps('obspy.plugin.waveform', 'writeFormat')
         for format in formats:
+            start = UTCDateTime(2009, 1, 13, 12, 1, 2, 999000)
             # XXX: skip SEGY and SU formats for now as they need some special
-            # headers.
-            if format in ['SEGY', 'SU', 'SEG2']:
+            # headers. Also skip GCF as format does not permitt fractional
+            # start time for sampling rates < 250
+            if format in ['SEGY', 'SU', 'SEG2', 'GCF']:
                 continue
+            elif format in ['GCF']:
```

Comment:
This `elif` will not be reached, since GCF is added above in the block that just does a `continue`.

These checks for `GCF` (also a few more cases above) would better be rewritten as `elif format == 'GCF':`, I think, but thats just cosmetics

Reply:
Yup, adjusted, the reason for `format in [...]` is mainly to pave the way for other formats with similar requirements in the future (e.g. the bc format I plan to add once I've figured out how to finish the current project)